### PR TITLE
Fix inconsistency in isEvenInner example

### PR DESCRIPTION
**Pull request by mieszko4**

### CODE REVIEW

File: docs/underscore.util.trampolines.js.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ isEvenNaive(99999);
 function isEvenInner (num) {
     if (num === 0) return _.done(true);
     if (num === 1) return _.done(false);
-    return function () { return isEvenInner(Math.abs(num) - 2); };
+    return function () { return isEvenInner(num - 2); };
```
Comment (jgonggrijp) [inline docs/underscore.util.trampolines.js.md:41]: Arguably, the `Math.abs` is correct here, but it was forgotten on line 32.

Comment (mieszko4) [reply docs/underscore.util.trampolines.js.md:41]: thnx, it is ok, you can close that pr and fix it on your preference.

Comment (jgonggrijp) [reply docs/underscore.util.trampolines.js.md:41]: Sure. I think I'll still do it in a continuation of your branch though. You discovered the inconsistency, so I'd prefer to give you the credit for that.

Comment (mafiosso) [reply docs/underscore.util.trampolines.js.md:41]: I vote for adding ```Math.abs``` to the first motivation example - negative numbers as input leads to infinite recursion.

**Pull request merged**